app/components/provider_interface/report_table_component.rb

```diff
@@ -2,12 +2,13 @@ module ProviderInterface
   class ReportTableComponent < ViewComponent::Base
     include ViewHelper
 
-    attr_reader :headers, :rows, :show_footer
+    attr_reader :headers, :rows, :show_footer, :ignore_last_footer_column
```

Comment:
Just about to change this! Instead it will be `columns_to_exclude_from_footer` or similar. Rather than excluding just the last column, you will be able to specific which columns you don't want a total for